### DISCUSSION TITLE
Is there a way to open response in browser?

### DISCUSSION THREAD

**Discussion by gachikuku** [mitmproxy/mitmproxy#7238] [Q&A] [answered]
When I replay a request and I get the output I was looking for, is there a way for me to open it in the browser?

**Comment by emanuele-em** [comment]
The question is not entirely clear to me, but I suggest using _addons_. You can run mitmproxy with a custom Python script to inspect the response and perform actions if it matches what you're searching for. 

Check out the [addons documentation](https://docs.mitmproxy.org/stable/addons-overview/) for more details, and here's an [example](https://docs.mitmproxy.org/stable/addons-examples/#log-events) that might give you some guidance 🙂

**Reply by gachikuku** [reply, marked as answer]
From what I see, probably it's probably not implemented (Show response in browser).
